models/members.js

```diff
@@ -23,8 +26,8 @@ const createModel = (sequelize, DataTypes) => {
     updatedAt: false
   })
   Members.associate = function (models) {
-    Members.belongsTo(models.Groups, { foreignKey: 'id_group', as: 'groups' })
+    Members.belongsTo(models.Groups, { foreignKey: 'id_group', as: 'group' })
```

Comment:
Members - Groups has **1 - many** relationship so use singular 'group' instead of 'groups' will generate a `getGroup` method instead of `getGroups` one.